boost-maven/boost-maven-plugin/src/it/test-jaxrs-2.0/src/test/java/it/LibertyFeatureVersionIT.java

```diff
@@ -23,17 +23,23 @@
 
     private static final String JAXRS_20_FEATURE = "<feature>jaxrs-2.0</feature>";
     private static String SERVER_XML = "target/liberty/wlp/usr/servers/defaultServer/server.xml";
+    private static String MESSAGES_LOG = "target/liberty/wlp/usr/servers/defaultServer/logs/messages.log";
+    private static String runtimeVersion;
+    private static String defaultRunTimeVersion = "19.0.0.9";
 
     @BeforeClass
     public static void init() {
         String runtime = System.getProperty("boostRuntime");
         org.junit.Assume.assumeTrue("ol".equals(runtime) || "wlp".equals(runtime));
+        runtimeVersion = System.getProperty("boost_runtime_version");
     }
 
     @Test
     public void testLibertyFeatureVersion() throws Exception {
         File targetFile = new File(SERVER_XML);
         assertTrue(targetFile.getCanonicalFile() + "does not exist.", targetFile.exists());
+        if (runtimeVersion == null)
+            runtimeVersion = defaultRunTimeVersion;
```

Comment:
Hmm...I'm a bit wary about saying this is part of the IT API (that you get a "boost_runtime_version" passed as a sysprop).    If we do this I'm OK.. but let's only do so explicitly...so let's remove the default here so that it blows up if we fail to provide this property... instead of having logic buried in the test here.